packages/client/src/components/Answers.tsx

```diff
@@ -4,52 +4,51 @@ import React, { FunctionComponent } from "react";
 import { FieldErrors } from "react-hook-form";
 
 import { ComponentWrapper, Label } from "../atoms";
-import { useTopicData } from "../hooks";
-import { Answer } from "../types";
+import { Answer, AnswerValue } from "../types";
 import { booleanOptions } from "../utils";
 import { QUESTION_ANSWERS } from "../utils/test-ids";
 
 type AnswersProps = {
+  answer?: AnswerValue;
   errors: FieldErrors; // This prop needs to be passed down, because the useForm() hook fails fetching `errors` in this component
-  question: imtr.Question;
+  question?: imtr.Question;
```

Comment:
Thanks for pointing this out, I will add to documentation. `question` is indeed optional, because a pre-question is a client question and not an IMTR Question. That's why `questionId` is required.